.eslintignore

```diff
@@ -0,0 +1,23 @@
+_cdn
```

Comment:
I believe we only run eslint on files under docs/src/js. Instead of doing all of these ignores, is it possible to do the inverse and just configure the path we want to run on?

Reply:
honestly the easiest thing to do might be to run eslint on some other process and not lint-staged.

Eslint doesn't seem to work very well with lint-staged, because you're right, previously the script had eslint run on docs/src/js and it worked fine.

Reply:
Estlint was running on the push, right?

Reply:
We could just move the eslint script out of lint-staged and have eslint run as part of the `npm start` and `npm build` scripts